draco/synthesis/stream.py

```diff
@@ -352,3 +352,73 @@ def _next_time_axis(self):
         self._cur_time += nsamp * int_time
 
         return time
+
+
+class MakeSiderealDayStream(task.SingleTask):
```

Comment:
just wondering if there is a better name for this task... Maybe just SiderealDayStream or SiderealStreamLSD. But that's up to you Richard.

Reply:
Yeah. I couldn't think of a better one. This is vaguely in line with our conventions for other names. At somepoint we might go through all the task names and try and impose some order on them.